client/components/Models.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { fetchProducts } from '../store/redux/products';
+import Model from './Model';
+
+class Models extends React.Component {
+  componentDidMount() {
+    this.props.getProducts();
+  }
+
+  render() {
+    console.log(this.props);
+    return (
+      <div>
+        {this.props.products.length ? (
```

Comment:
While we do want this check to happen (to make sure that we're properly handling what happens when we're out of cars) it looks like it's muddying the rest of the HTML right now, being a little hard to interoperate. It might be worth thinking about if we can move this check somewhere else or rephrase it in a way that'll save us some indentation